CHANGES.md

```diff
@@ -1,3 +1,11 @@
+# fm 0.1.1 (2020-07-08)
+
+* Add `ignore_leading_whitespace` and `ignore_leading_whitespace` options. Both
```

Comment:
The second backticks should be `ignore_trailing_whitespace`.

Reply:
Fixed in https://github.com/softdevteam/fm/pull/15/commits/d4c4643e512c42b23fc5e996c579050fd94ac1ed.